lib/src/util/context.dart

```diff
@@ -135,34 +137,34 @@ class Context<TValue> {
 ///
 /// See: <https://reactjs.org/docs/context.html#contextprovider>
 class ProviderProps<TValue> extends builder_helpers.UiProps {
-  ProviderProps(JsBackedMap? backingMap)
-      : this._props = JsBackedMap() {
-    this._props = backingMap ?? JsBackedMap();
-  }
+  ProviderProps([Map? backingMap]) : this.props = backingMap ?? JsBackedMap();
```

Comment:
Should the fallback be `const {}` now - or is it still `JsBackedMap` intentionally?

Reply:
It can't be `const` since that would make the map read-only, and throw when setting props on it. JsBackedMap is the default empty map used in generated implementations, and also in other manually-implemented UiProps constructors, since it's a little more optimal when copying props during ReactElement creation.

Generated example: https://github.com/Workiva/over_react/blob/5c729fd764777e4c0fa29809853d82e2afa518b1/test_fixtures/gold_output_files/mixin_based/basic.over_react.g.dart.goldFile#L23C1-L25
Manually-authored example: https://github.com/Workiva/over_react/blob/5c729fd764777e4c0fa29809853d82e2afa518b1/lib/src/component/dom_components.dart#L47